### DISCUSSION TITLE
Free options where it can be released to production...

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#328] [Q&A]
I want to do a small project for my portfolio and I would like to display my pocket for free, thanks for your help.

**Comment by ganigeorgiev** [comment]
I haven't tested it yet, but Fly supports mounting persistent volumes, so you should be able to use their free tier - https://fly.io/docs/about/pricing/

**Reply by jossmanbr2022** [reply]
**Thanks,** I'll try this option and if it works for me, I'll comment here.

**Reply by katriellucas** [reply]
I would like to know how you did it, if you can make it work.

**Reply by aleda145** [reply]
I did a lab with pocketbase on fly.io with a react SPA frontend. I can clean it up and paste a repo link here :smile:

**Comment by aleda145** [comment]
Here is my repo where I host pocketbase for free on fly.io

https://github.com/aleda145/pocketbase-lab 

I'm also  hosting the static front end for free on Cloudflare Pages
Hope it helps!

**Reply by katriellucas** [reply]
Thank you, i'm sure that will help.

**Comment by ganigeorgiev** [comment]
@katriellucas, @jossmanbr2022 You can find an example guide how to deploy for free on fly.io here - https://github.com/pocketbase/pocketbase/discussions/537

**Reply by katriellucas** [reply]
Thank you @ganigeorgiev, I had figured it myself using @aleda145 example, with that I was able to deploy it not only on fly.io but on [koyeb](https://www.koyeb.com/) and [mogenius](https://mogenius.com/) as well.
